users/management/commands/remove-duplicate-siap-users.py

```diff
@@ -14,13 +14,20 @@ def find_standalone_users_with_siap_account():
         User.objects.values("email")
         .annotate(email_count=Count("email"))
         .filter(email_count__gt=1)
+        .exclude(is_superuser=True)
+        .exclude(is_staff=True)
```

Comment:
Tip : on peut écrire les exclude en une fois avec plusieurs kwargs 

```suggestion
        .exclude(is_superuser=True, is_staff=True)
```

Reply:
j'étais pas sur si ça faisait un ou ou un et :)